migrations/migrations/20210406205811-name-of-migration.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+      return queryInterface.createTable('user', {
+        id: {
+          type: Sequelize.INTEGER,
+          unique: true,
+          autoIncrement: true,
+          primaryKey: true,
+          allowNull: false
+        },
+        firstName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        lastName: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        email: {
+          type: Sequelize.STRING,
+          unique: true,
+          allowNull: false
+        },
+        password: {
+          type: Sequelize.STRING,
+          allowNull: false
```

Comment:
please verify that the timestamps columns are being created `created_at` & `updated_at`

Reply:
timestamp fields added